security/pod-to-pod-encryption.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Enable pod-to-pod encryption
+description: Enable WireGuard for state-of-the-art cryptographically security between pods for Calico Enterprise clusters.
+---
+
+### Big picture
+
+Enable WireGuard to secure pod-to-pod communication in a Calico Enterprise cluster.
```

Comment:
In body of the doc, please use {{site.prodname}} throughout. (Do not use in title or description because it won't render).